x-pack/test/security_solution_cypress/es_archives/risk_scores_new/data.json

```diff
@@ -483,3 +483,504 @@
     }
   }
 }
+
+
+{
```

Comment:
Add the same data to the data stream index.

Reply:
What data is this the same as? And why is it being added?

Reply:
Previously, the file only inserted documents into the latest index, which resulted in the time-series index being empty during tests. However, after my change, it will now insert the same documents into both indices. So, any features that rely on the time-series index will now function as expected.